present/miniPresent.py

```diff
@@ -4,20 +4,18 @@
 
 
 class MiniPresent:
+    # import string
     def __init__(self, key, rounds=4):
         self.rounds = rounds
-        if isinstance(key, basestring):
-            self.roundkeys = generateRoundkeys16(string2number(key), self.rounds)
-        else:
-            self.roundkeys = generateRoundkeys16(key, self.rounds)
+        self.roundkeys = generateRoundkeys16(string2number(key), self.rounds)
```

Comment:
I feel like this change makes the code more cluttered while you try to support both strings and bytes in a method that clearly indicates it should work with strings.